sklearn/calibration.py

```diff
@@ -75,6 +104,32 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin,
         .. versionchanged:: 0.22
             ``cv`` default value if None changed from 3-fold to 5-fold.
 
+    n_jobs : int, default=None
+        Number of jobs to run in parallel.
+        ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
+        ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
+        for more details.
+
```

Comment:
And the rest of the new parameters.

```suggestion

        .. versionadded:: 0.24
        
```

Reply:
I guess this change should come just before `n_jobs` in the doc-string, right?

Reply:
Ah nevermind, I just have understood the syntax.